src/topology/metric_space/basic.lean

```diff
@@ -196,6 +196,10 @@ variables [pseudo_metric_space α]
 instance metric_space.to_uniform_space' : uniform_space α :=
 pseudo_metric_space.to_uniform_space
 
+@[priority 100] -- see Note [lower instance priority]
+instance pseudo_metric_space.to_bornology' {α : Type u} [pseudo_metric_space α] : bornology α :=
+pseudo_metric_space.to_bornology
```

Comment:
I copied from `to_uniform_space'`. I don't know whether TC search can see through default arguments. If it can, then we can use something like `attribute [priority 100, instance] pseudo_metric_space.to_uniform_space pseudo_metric_space.to_bornology`.

Reply:
That's convincing enough for me, let's tidy this in another PR

Reply:
Should we merge it then, or you did not review the rest of the PR?

Reply:
I'm not really failiary enough with bornologies to review the rest, sorry

Reply:
Done in #14639